config/locales/en-GB.yml

```diff
@@ -225,3 +225,18 @@
 
       full_messages:
         format: "%{attribute} %{message}"
+
+  mailer:
```

Comment:
I think we've only been using the `en-US`, not `en-GB`, locale with the emails, so this can be removed unless Translation specifically requested it.